src/rows.c

```diff
@@ -51,22 +51,31 @@ size_t compute_row_size(int actual_num_fields,
 }
 
 
-PyObject *call_converter_function(PyObject *func, char32_t *token)
+PyObject *call_converter_function(PyObject *func, char32_t *token, bool byte_converters)
 {
     Py_ssize_t tokenlen = 0;
     while (token[tokenlen]) {
         ++tokenlen;
     }
+
+    // Convert token to a Python unicode object
     PyObject *s = PyUnicode_FromKindAndData(PyUnicode_4BYTE_KIND, token, tokenlen);
     if (s == NULL) {
-        // fprintf(stderr, "*** PyUnicode_FromKindAndData failed ***\n");
         return s;
     }
+
+    // If byte_converters flag is set, encode prior to calling converter
+    if (byte_converters) {
+        // TODO: Is it safe to use the same PyObject to store the encoded result?
+        s = PyObject_CallMethod(s, "encode", NULL);
```

Comment:
Probably OK to do it like this, but it should use `PyUnicode_AsEncodedString`, calling into python like this is just a pain (and slow).

Reply:
One other thing, this should use `"latin1"` as encoding parameter, I would _hope_ some test fails if you don't do it...

Reply:
Good catch, I missed the latin1 thing from np.loadtxt. Unfortunately this leads to a `MemoryError` so something is clearly going wrong somewhere. I'll need to dig deeper to figure out what's happening...

Reply:
A `MemoryError`?  Sounds strange, so I bet it is just some typo-style bug :).